README.md

```diff
@@ -61,12 +61,16 @@ Make sure to install the Hugo extended version specified by the `HUGO_VERSION` e
 To install dependencies, deploy and test the site locally, run:
 
 - For macOS and Linux
+  
   ```bash
   npm ci
   make serve
   ```
 - For Windows (PowerShell)
+  
   ```powershell
+  Remove-Item -Path "i18n" -Recurse -Force
+  $CD = Get-Location; New-Item -ItemType Junction -Path "$CD\i18n" -Value "$CD\data\i18n" 
```

Comment:
We'd need to warn people not to commit the change they make here. Is there any alternative that doesn't involve running `Remove-Item`?

Reply:
Maybe a PowerShell one liner to convert the symlinks into something Windows native?

Reply:
I'm currently looking into converting the symlinks into something more native to Windows using PowerShell. I appreciate your patience as I work on finding the solution. I'll make sure to update you as soon as I have it sorted out

Reply:
Hey @sftim , i have just tested deploying the site locally , found out that we can now easily access the site locally without any issue on Windows . I think this problem is resolved as we upgraded to node version 20.11.0 and hugo version 0.121.2 . When i made this PR at that time we are using other version of node and hugo .So in my opinion it is good to close this PR .

Reply:
Feel free to close your PR!